src/lib/SX1280Driver/SX1280.cpp

```diff
@@ -0,0 +1,344 @@
+#include "SX1280_Regs.h"
+#include "SX1280_hal.h"
+#include "SX1280.h"
+
+SX1280Hal hal;
+/////////////////////////////////////////////////////////////////
+SX1280Driver *SX1280Driver::instance = NULL;
+//InterruptAssignment_ InterruptAssignment = NONE;
+
+//uint8_t SX127xDriver::_syncWord = SX127X_SYNC_WORD;
+
+//uint8_t SX127xDriver::currPWR = 0b0000;
+//uint8_t SX127xDriver::maxPWR = 0b1111;
+
+/* Steps for startup 
+
+1. If not in STDBY_RC mode, then go to this mode by sending the command:
+SetStandby(STDBY_RC)
+
+2. Define the LoRa® packet type by sending the command:
+SetPacketType(PACKET_TYPE_LORA)
+
+3. Define the RF frequency by sending the command:
+SetRfFrequency(rfFrequency)
+The LSB of rfFrequency is equal to the PLL step i.e. 52e6/2^18 Hz. SetRfFrequency() defines the Tx frequency.
+
+4. Indicate the addresses where the packet handler will read (txBaseAddress in Tx) or write (rxBaseAddress in Rx) the first
+byte of the data payload by sending the command:
+SetBufferBaseAddress(txBaseAddress, rxBaseAddress)
+Note:
+txBaseAddress and rxBaseAddress are offset relative to the beginning of the data memory map.
+
+5. Define the modulation parameter signal BW SF CR
+*/
+
+uint32_t beginTX;
+uint32_t endTX;
+
+void ICACHE_RAM_ATTR SX1280Driver::nullCallback(void){return;};
```

Comment:
Extra semicolon on the end here isnt needed. Would also be nice to have a space in between the round and curly braces.